### PR TITLE
feat: citadel banner [WEB-1495]

### PR DESCRIPTION
**Pull request by pentcle**
1. add citadel banner using existing banner code (could not use web-lib as missing dep framer-motion)
2. added min-height as there isn't enough copy to push the banner image to the desired height
3. added .npmrc to .gitignore (the commenting out is probably enough magic but didn't want to risk pushing my pat)
4. also to note i installed the prettier package from the codebase and it reformatted a couple of lines of code in the vaults file - hope that's ok

image

image

### CODE REVIEW

File: src/client/routes/Vaults/index.tsx [hunk 1]
```diff
@@ -207,21 +211,13 @@ export const Vaults = () => {
   return (
     <ViewContainer>
       <StyledSliderCard
-        header={t('vaults:your-time-card.header')}
+        header="Goblin town or the Citadel?"
```
Comment (karelianpie) [inline src/client/routes/Vaults/index.tsx:214]: Should this be added to the translation file `public/locales/en/vaults.json`?

Comment (pentcle) [reply src/client/routes/Vaults/index.tsx:214]: yes - is it possible to do that as a second task? the socials for this just went live

File: src/client/routes/Vaults/index.tsx [hunk 2]
```diff
@@ -207,21 +211,13 @@ export const Vaults = () => {
...
           </Text>
         }
-        // NOTE Example for slideshow array
-        // slidesContent={[
-        // {
-        //   header: 'Test header',
-        //   content: 'test content',
-        // },
-        // ]}
+        background={<img src={GoblinTown} alt="Goblin town or the Citadel?"></img>}
```
Comment (karelianpie) [inline src/client/routes/Vaults/index.tsx:220]: nit: We don't need to close an `img` tag
```suggestion
        background={<img src={GoblinTown} alt="Goblin town or the Citadel?">}
```

File: src/client/routes/Vaults/index.tsx (hunk 1 above)
Comment (karelianpie) [reply src/client/routes/Vaults/index.tsx:214]: Yes, all good 👍

**Review by karelianpie — APPROVED** [review]
LGTM, just a couple of minor comments that can be addressed in a different PR

**Comment by pentcle** [comment]
> LGTM, just a couple of minor comments that can be addressed in a different PR

just pushed again - the suggested change didn't have the image tag closing slash:  />

**Comment by karelianpie** [comment]
> just pushed again - the suggested change didn't have the image tag closing slash: />

That's not necessary

**Review by xgambitox — COMMENTED** [review]
@pentcle can you try reverting changes of `yarn.lock` file, running `yarn install` again, and pushing changes? I ran `yarn install` in develop branch and just resulted in minor changes, so lets see if that helps with the build

**Review by karelianpie — APPROVED** [review]

**Review by xgambitox — APPROVED** [review]

**Review by turtlemoji — APPROVED** [review]

**Pull request merged**